src/main/java/com/itrex/java/lab/controller/TestDatabaseFillingController.java

```diff
@@ -0,0 +1,46 @@
+package com.itrex.java.lab.controller;
+
+import com.itrex.java.lab.service.FillingDatabaseTestService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/add")
+@RequiredArgsConstructor
+public class TestDatabaseFillingController {
+
+    private final FillingDatabaseTestService service;
+
+    @PostMapping("/users")
+    @PreAuthorize("hasAuthority('database:fill')")
+    public ResponseEntity addUsers(@RequestParam(name = "role") String role,
```

Comment:
For POST method use RequestBody instead of RequestParams

Reply:
RequestParams deleted.